tests/Helper/ResponseHelper.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright (c) 2013-2017 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+
+namespace OpenCFP\Test\Helper;
+
+use Symfony\Component\HttpFoundation;
+
+/**
+ * @mixin \PHPUnit\Framework\TestCase
+ */
+trait ResponseHelper
+{
+    final protected function assertResponseIsSuccessful(HttpFoundation\Response $response)
+    {
+        $this->assertTrue($response->isSuccessful(), 'Failed asserting that the response is successful.');
+    }
+
+    final protected function assertResponseIsRedirect(HttpFoundation\Response $response)
```

Comment:
About `assertResponseIsSuccessful()` and `assertResponseIsRedirect()`:

Personally, I would remove them. They are not specific enough. For example, we probably want to assert that a redirect is permanent, or temporary. Same for successful responses, we probably want to be more specific as well and assert the *actual* status code.

What do you think?

Reply:
I like them. Unless I'm building a JSON API, I'm not sure I find myself writing specs that specific. For typical web interactions, I mostly just want to know "they were redirected to the correct endpoint after submission". I'm not sure I care `301` vs `302` and might see that as over-specification.

What if we had assertions per response class (`2xx`, `3xx`, `4xx`, `5xx`) with the option of passing in a second argument for specific response code? It's just syntactic sugar, but I think that's important for readability. 

Case in point, `assertResponseStatus(Response::HTTP_OK, $response)` hurts my eyes more than `assertResponseIsSuccessful($response)`.